.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+# Fixes # 
+
+### Checklist
+- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar).
+- [ ] I have read the [Contribution Guide](CONTRIBUTING.md) and my PR follows them.
+- [ ] I have discussed these changes via issue.
+- [ ] I have made my changes on a new branch.
+- [ ] My code follows the code style of this project.
+- [ ] I have updated the documentation accordingly.
```

Comment:
We don't currently have documentation or tests, so I don't think there's a need for that. We could remove that line and maybe bring it back once #7 is closed.
```suggestion

```